mini-modeline.el

```diff
@@ -80,8 +80,28 @@ Will be set if `mini-modeline-enhance-visual' is t."
   :type 'boolean
   :group 'mini-modeline)
 
+(defface mini-modeline-active-window-modeline
```

Comment:
nit: To consistent with original face `mode-line`, I think we should change the name of this to `mini-modeline-mode-line` and the one bellow to `mini-modeline-mode-line-inactive`.

Reply:
Done.